neuralop/layers/discrete_continuous_convolution.py

```diff
@@ -764,6 +813,7 @@ def __init__(
         in_shape: Tuple[int],
         out_shape: Tuple[int],
         kernel_shape: Union[int, List[int]],
+        filter_basis: str="piecewise_linear",
```

Comment:
shall we call this `basis_type` for consistency with torch-harmonics?

Reply:
Sure